deployment/MIGRATIONS.md

```diff
@@ -63,7 +63,7 @@ $ ssh 10.0.1.191
 After successfully connecting to an application server, run `setupdb.sh`:
 
 ```bash
-$ /opt/nyc-trees/scripts/aws/setupdb.sh
+$ DJANGO_SITE_DOMAIN="treescount.foo.com" /opt/nyc-trees/scripts/aws/setupdb.sh
```

Comment:
No, just this piece of documentation, and even that is not required. When I first commented, I missed that it was documentation and thought it was a placeholder value.